### ISSUE TITLE
Getting TypeError when using development version of React with cellRenderer

### ISSUE THREAD

**Issue by Martynas-P** [facebook/fixed-data-table#154]
I'm getting the following error

```
TypeError: Cannot read property 'hasOwnProperty' of undefined
```

when I use Fixed Data Table with development version of React and have a `cellRenderer` property for one of the columns specified. The error does not happen when I use the production version of React.

React: 0.13.3
Fixed Data Table: 0.2.0

JS Code: https://gist.github.com/Martynas-P/e343680cad50d1c791f4

**Comment by nbostrom** [comment]
Probably related to #146

**Comment by rundis** [comment]
Same problem here.
Fully reproducible example:
https://gist.github.com/rundis/145cfdaf6ea277c9647b

**Comment by Martynas-P** [comment]
Seems to work now in 0.3 with React 0.13.3

**Comment by rundis** [comment]
Are you sure ? 
My example is with  fixed-data-table 0.3 and React 0.13.3, and it certainly doesn't work.
Check out the gist. Can you run that without any problem ?

cheers
rundis

**Comment by ehzhang** [comment]
Hey @rundis , I ran your gist, and encountered an error:
If you change line 18 to the most up to date release, it works fine.

I've attached a gist here:
https://gist.github.com/ehzhang/daf4453df43a4dede3d8

![image](https://cloud.githubusercontent.com/assets/3484848/8093517/747d3166-0f76-11e5-9170-d05e2bcc0883.png)
